src/network/networkd-wwan.h

```diff
@@ -0,0 +1,85 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+#pragma once
+
+#include <inttypes.h>
+#include <stdbool.h>
+
+#include "sd-bus.h"
+
+#include "in-addr-util.h"
+#include "network-util.h"
+#include "networkd-wwan-bus.h"
+
+typedef struct Link Link;
+typedef struct Manager Manager;
```

Comment:
Please drop all header inclusion, and include only `networkd-forward.h` and `networkd-wwan-bus.h`. Then, drop redefinitions of Link and Manager.